lib/trino-hive-formats/src/main/java/io/trino/hive/formats/encodings/binary/BinaryColumnEncodingFactory.java

```diff
@@ -23,8 +23,8 @@
 import io.trino.spi.type.TimestampType;
 import io.trino.spi.type.Type;
 import io.trino.spi.type.VarcharType;
-import org.joda.time.DateTimeZone;
 
```

Comment:
https://github.com/trinodb/trino/pull/18632#issuecomment-1673332891
>Did you measure the impact in performance? The reason we originally didn’t remove joda everywhere is that the Java datetime APIs require some usage patterns that can be inefficient.

@martint I've run benchmarks which @raunaqmorarka adviced.
https://github.com/trinodb/trino/pull/18632#discussion_r1291163921
So far I see performance improvement with this change.
I'd be glad to reuse knowledge gathered during previous attempts. 
Please point me where previous attempts took place. 
Could you please elaborate what are those inefficient usage patterns you are mentioning?

Reply:
@martint I see, you have concerns about additional objects creation and how JIT optimizes execution flows.
From the first glance joda has more math and less of objects creation (however still has).
Running tests provided by @raunaqmorarka show it's not a problem. 
Also those tests prove java.time is much faster than joda.

I'm more than happy to run tests and be sure that my changes do not bring regression, especially in performance.
However, I'm new to performance testing in trino. 
How is it usually done? Where I can find documentation about it? When/by whom usually those test run? 
Where I can find performance tests which justify joda presence?
What is the acceptance criteria for this PR?

Also I see that this PR may be more controversial that I anticipated and require more opinions to gather.
Could you please help me identify involved contributors/maintainers who has worked on the initial partial migration from joda to java.time or who can be interested in this topic? Should I move the discussion to the slack according to https://trino.io/development/process.html ?

I'm eagerly waiting for your answer as I really want to have this PR merged or to have a prove that it does not worth.